The build system includes its own check for available CMake build types.
This check is incorrect as it doesn't include the None build type.

diff -upr keystone-0.9.1.orig/llvm/CMakeLists.txt keystone-0.9.1/llvm/CMakeLists.txt
--- keystone-0.9.1.orig/llvm/CMakeLists.txt	2020-02-22 12:17:15.681580953 +0100
+++ keystone-0.9.1/llvm/CMakeLists.txt	2020-02-22 12:18:20.949144210 +0100
@@ -142,7 +142,7 @@ include(CPack)
 string(TOUPPER "${CMAKE_BUILD_TYPE}" uppercase_CMAKE_BUILD_TYPE)
 
 if (CMAKE_BUILD_TYPE AND
-    NOT uppercase_CMAKE_BUILD_TYPE MATCHES "^(DEBUG|RELEASE|RELWITHDEBINFO|MINSIZEREL)$")
+    NOT uppercase_CMAKE_BUILD_TYPE MATCHES "^(NONE|DEBUG|RELEASE|RELWITHDEBINFO|MINSIZEREL)$")
   message(FATAL_ERROR "Invalid value for CMAKE_BUILD_TYPE: ${CMAKE_BUILD_TYPE}")
 endif()
 
